recipes/nvim/build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# libtool produces hardcod path! https://github.com/conda-forge/libtool-feedstock/issues/18
```

Comment:
Please use the standard toolchain

Reply:
@scopatz Thanks for taking look. `libtool` from conda-forge is broken for a long time as in [this issue](https://github.com/conda-forge/libtool-feedstock/issues/18). It contains the hardcoded path in build time if you pull the package. Linux build env has libtool by default I'm not 100% sure. However, Mac build env doesn't have glibtool (g means GNU version, which is usually got from `brew install`). I can remove for Linux if it has libtool in std but Mac might have trouble. So question would be: does conda azure Mac build env has glibtool by default?

Reply:
Can you give a link to the failing log?

Reply:
So that recipe probably needs prefix replacement on it, if it doesn't have it already. I think it is this field: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#binary-relocation

Alternatively, don't package for mac for the moment

Reply:
It looks like it's confused due to the paths to tools like `x86_64-conda-linux-gnu-as` being in the build environment so prefix replacement won't work. It should be easy to fix if @daizeng1984 can provide a demonstration of it failing.

Reply:
Thanks for replies. Right, replacement configs in meta.yaml doesn't work for libtool. [Here](https://gist.github.com/daizeng1984/b206573a0f4a380de0be6a9a07c326d0)'s failing log on my Mac removing the libtool. The error line should be this one:
```
error: $BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-libtool: unknown option character `-' in: --quiet
```
[Here](https://gist.github.com/daizeng1984/79fabf6b15f9c89e62e719c3fb39eb97)'s the failing log using conda forge libtool and error lines:
```
$BUILD_PREFIX/bin/libtool: line 1734: /Users/runner/miniforge3/conda-bld/libtool_1596899629849/_build_env/bin/x86_64-apple-darwin13.4.0-ar: No such file or directory
```
BTW, these tools are needed only for build not runtime.

Reply:
Will be fixed by https://github.com/conda-forge/libtool-feedstock/pull/22